meson_options.txt

```diff
@@ -482,6 +482,8 @@ option('fuzz-tests', type : 'boolean', value : false,
        description : 'run the fuzzer regression tests by default (with sanitizers)')
 option('install-tests', type : 'boolean', value : false,
        description : 'install test executables')
+option('unittestsdir', type : 'string', value : '',
+       description : 'name of the subdirectory of the unit tests')
```

Comment:
Hmm, we have `bashcompletiondir` and `zshcompletiondir` and so on (without "s"). And for new names, we generally use punctuation. Can you call this `unittest-dir` for readability?

OTOH, is it even worth making this configurable? This will make distros differ with no good reason. I wouldn't mind just installing the tests to `testsdir / 'unittests'` unconditionally.

Reply:
> OTOH, is it even worth making this configurable? This will make distros differ with no good reason. I wouldn't mind just installing the tests to testsdir / 'unittests' unconditionally.

It breaks a bunch of stuff otherwise

Reply:
I agree: having a build option "just" to avoid some stuff breaking seems awkward but I didn't know exactly what needs to be adjusted and how much work was involved to cope with this change.